packages/neutrino-preset-web/index.js

```diff
@@ -85,8 +84,10 @@ module.exports = (neutrino) => {
     .when(process.env.NODE_ENV !== 'test', () => neutrino.use(chunk))
     .target('web')
     .context(neutrino.options.root)
+	 .entry('vendor')
+      .add(require.resolve('./polyfills.js'))
```

Comment:
Removing the polyfills file different, since we can't just remove based on the location of babel-polyfill. I'm not saying this is a bad thing, just different. We should probably add this to the docs, about how to remove the polyfills.

```js
neutrino.config
  .entry('vendor')
    .delete(require.resolve('neutrino-preset-web/polyfills'))
```